packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,38 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [33.0.0]
+
+### Added
+
+- Added `Mutex` lock in the `updateNftMetadata` function ([#4325](https://github.com/MetaMask/core/pull/4325))
+- Token price API support for mantle network ([#4376](https://github.com/MetaMask/core/pull/4376))
+
+### Changed
+
+- **BREAKING:** `NftController` constructor argument `selectedAddress` has been removed. ([#4221](https://github.com/MetaMask/core/pull/4221))
+- **BREAKING:** `NftController` now requires `AccountsController:get{Account,SelectedAccount}` messenger actions. ([#4221](https://github.com/MetaMask/core/pull/4221))
+- **BREAKING:** `NftController` now requires `AccountsController:selectedEvmAccountChange` event. ([#4221](https://github.com/MetaMask/core/pull/4221))
+- **BREAKING:** `NftDetectionController` now requires `AccountsControlelr:getSelectedAccount` messenger actions. ([#4221](https://github.com/MetaMask/core/pull/4221))
+- **BREAKING:** `AccountTrackerController` now requires a controller messenger ([#4225](https://github.com/MetaMask/core/pull/4225))
+- **BREAKING:** Removed `getIdentities`, `getSelectedAddress` and `onPreferencesStateChange` from the constructor arguments of the `AccountTrackerController` ([#4225](https://github.com/MetaMask/core/pull/4225))
+- **BREAKING:** Bump dependency and peer dependency `@metamask/accounts-controller` to `^17.0.0` ([#4413](https://github.com/MetaMask/core/pull/4413))
+- Upgrade `TokenRatesController` to `BaseControllerV2` ([#4314](https://github.com/MetaMask/core/pull/4314))
+- Update Nft controllers to use `selectedAccountId` instead of `selectedAddress` ([#4221](https://github.com/MetaMask/core/pull/4221))
+- `TokenRatesController` uses checksum instead of lowercase format for token addresses ([#4377](https://github.com/MetaMask/core/pull/4377))
+- Bump `@metamask/keyring-api` to `^8.0.0` ([#4405](https://github.com/MetaMask/core/pull/4405))
+- Bump `@metamask/eth-snap-keyring` to `^4.3.1` ([#4405](https://github.com/MetaMask/core/pull/4405))
+- Bump `@metamask/keyring-controller` to `^17.1.0` ([#4413](https://github.com/MetaMask/core/pull/4413))
+
+### Removed
+
+- Remove `value` from price API responses ([#4364](https://github.com/MetaMask/core/pull/4364))
+
+### Fixed
+
+- Catch the case when data are not available for given token ([#4361](https://github.com/MetaMask/core/pull/4361))
```

Comment:
`catch` is an implementation detail that can be omitted from the changelog. I think we could be a bit more clear on what is being made newly available to the user when market data is not returned by the codefi api.

Reply:
Also used the CHANGELOG section of this PR, maybe we could fallback to the PR's title in this case then, something like:

```
- Prevent the use of market data when not available for a given token
```

WDYT?

Reply:
Hmm based on the test case it looks like the PR might be doing the opposite:

```suggestion
- Fix token prices market data not being returned if price data is unavailable for any individual token ([#4361](https://github.com/MetaMask/core/pull/4361))
```

I'm not sure, though. @salimtb @bergeron Would you be able to provide context on this?

Reply:
hey everyone,
the goal of the PR is to revise the approach to ensure that when market data for token prices is not available, the corresponding row for that token is removed. This adjustment is made to handle instances where there is no available price conversion data for a specific token.
i agree with what @ccharly suggested we can use:
```
- Prevent the use of market data when not available for a given token
```

Reply:
thanks @salimtb, lemme use that then!

Reply:
@salimtb Thanks for the explanation!